### DISCUSSION TITLE
[question][typescript] ExecutionContext import

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [honojs/hono#1459] [Q&A] [answered]
Where can I find ExecutionContext type? I'm overriding the:

```ts
export default {
  fetch: (request: Request, env?: {}, executionCtx?: ExecutionContext) => {
    console.log('Server listening on port 5000...')
    return app.fetch(request, env, executionCtx)
  }
}
```

**Comment by EdamAme-x** [comment, marked as answer]
It can be imported from `https://deno.land/x/hono@v3.7.2/context.ts`.
```typescript
import type { ExecutionContext } from 'https://deno.land/x/hono@v3.7.2/context.ts';
```

**Reply by angelhdzmultimedia** [reply]
Thank you for that kind response.
I asked because I was using Hono in Bun and didn't find where to import `ExecutionContext`  from. 🤔
I also thought Hono was a Bun library so didn't ocurr to me to search in Deno land. 😊
